frontend/src/contexts/searchContext.tsx

```diff
@@ -93,6 +95,15 @@ export const schoolsOptions = Object.entries(schools).map(
   }),
 );
 
+const buildings = buildingData as Buildings[];
+
+export const buildingOptions = buildings.map(
+  (building): Option => ({
+    value: building.code,
+    label: building.building_name ?? building.code,
```

Comment:
I suggest doing `${building.building_name} (${building.code})` if the name exists, because the code is still useful in many contexts

Reply:
```suggestion
    label: building.building_name
      ? `${building.building_name} (${building.code})`
      : building.code,
```

(Formatting needed